src/scanner/Scanner.spec.ts

```diff
@@ -64,18 +66,73 @@ describe('Scanner', () => {
     });
   });
 
-  it('runs fix when fix flag set to true', async () => {
-    jest.setTimeout(20000);
-    const fixMock = jest.fn();
-    containerCtx = createTestContainer({ fix: true });
-    ESLintWithoutErrorsPractice.prototype.fix = fixMock;
-    ESLintWithoutErrorsPractice.prototype.evaluate = () => Promise.resolve(PracticeEvaluationResult.notPracticing);
-    containerCtx.container.bind('ESLintWithoutErrorsPractice').to(ESLintWithoutErrorsPractice);
-    const scanner = containerCtx.container.get(Scanner);
+  describe('fixer', () => {
+    jest.setTimeout(40000);
```

Comment:
Is it necessary to have so big timeout here? I'd remove the timeout.

Reply:
:wrench:  Fixed in d17c4435d2a8b06518aa39018ebfa3fabd3fbbc2